tests/units/anta_tests/test_system.py

```diff
@@ -346,6 +346,39 @@
         },
         "expected": {"result": "success"},
     },
+    {
+        "name": "success-ip-dns",
+        "test": VerifyNTPAssociations,
+        "eos_data": [
+            {
+                "peers": {
+                    "1.1.1.1 (1.ntp.networks.com)": {
```

Comment:
test will still pass if we have actual address as 1.1.1.11 and input address as 1.1.1.1

Reply:
Better if we can check the input address with peerIpAddr

Reply:
Good catch. Will fix it.

Reply:
Fixed in c8928bd0085d3e01d5fe77ca99065c22d71cae83